### PR TITLE
Add the scala survey 2023 announcement

### PR DESCRIPTION
**Pull request by KacperFKorban**
The instructions are pretty much the same as last year since the survey was intended to stay mostly the same as well.

### CODE REVIEW

File: blog/_posts/2023-09-21-scala-survey-2023-announcement.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+![Scala Survey 2023 Announcement]({{ site.baseurl }}/resources/img/scala-survey-2023-announcement.png)
+
+## The Scala Team wants to hear from you again!
+
+Over the last year we used your feedback from the Scala Survey and to improve the language we all love. This year we want to hear from you again. How has your usage of Scala changed over the last year? Which direction would you like for Scala to take? Take part in the Scala Survey 2023 and shape the future of Scala.
+
+[Scala Survey 2023](https://scala-survey-2023.virtuslab.com/)
+
+The Survey will close on **14.10.2023**.
```
Comment (SethTisue) [inline blog/_posts/2023-09-21-scala-survey-2023-announcement.md:16]:
```suggestion
The Survey will close on **October 14, 2023**.
```

File: blog/_posts/2023-09-21-scala-survey-2023-announcement.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## About the survey
+
+Completing the survey should take 4-12 minutes, and is anonymous.
+
+Your responses will help us understand Scala’s strengths and weaknesses. They will also be a major factor in determining the development priorities for the Scala team.
+
+We are interested in learning what libraries and tools people are using, what platforms they support, what are the painpoints in their everyday Scala usage, where people are learning to use scala/training engineers, etc.
+
+
+**Who is the survey intended for?** Everyone. Whether you’re an experienced Scala Developer or just starting your Scala journey, we count on your participation.
```
Comment (SethTisue) [inline blog/_posts/2023-09-21-scala-survey-2023-announcement.md:27]:
```suggestion
**Who is the survey intended for?** Everyone. Whether you’re an experienced Scala developer or just starting your Scala journey, we count on your participation.
```

File: blog/_posts/2023-09-21-scala-survey-2023-announcement.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+post-type: blog
+by: Kacper Korban, VirtusLab
+title: Scala Survey is back!
+---
+
+![Scala Survey 2023 Announcement]({{ site.baseurl }}/resources/img/scala-survey-2023-announcement.png)
+
+## The Scala Team wants to hear from you again!
+
+Over the last year we used your feedback from the Scala Survey and to improve the language we all love. This year we want to hear from you again. How has your usage of Scala changed over the last year? Which direction would you like for Scala to take? Take part in the Scala Survey 2023 and shape the future of Scala.
```
Comment (SethTisue) [inline blog/_posts/2023-09-21-scala-survey-2023-announcement.md:12]:
```suggestion
Over the last year we used your feedback from the Scala Survey to improve the language we all love. This year we want to hear from you again. How has your usage of Scala changed over the last year? Which direction would you like for Scala to take? Take part in the Scala Survey 2023 and shape the future of Scala.
```

**Review by SethTisue — CHANGES_REQUESTED** [review]

File: blog/_posts/2023-09-21-scala-survey-2023-announcement.md (hunk 1 above)
Comment (KacperFKorban) [reply blog/_posts/2023-09-21-scala-survey-2023-announcement.md]: Thanks, I forgot that people in the US switch the dates :sweat_smile:

Comment (SethTisue) [reply blog/_posts/2023-09-21-scala-survey-2023-announcement.md]: It is our national shame.

**Review by SethTisue — APPROVED** [review]

**Pull request merged**